sound/soc/sof/intel/bdw.c

```diff
@@ -633,13 +633,14 @@ const struct snd_sof_dsp_ops sof_bdw_ops = {
 			SNDRV_PCM_INFO_PAUSE |
 			SNDRV_PCM_INFO_BATCH,
 };
-EXPORT_SYMBOL(sof_bdw_ops);
+EXPORT_SYMBOL_NS(sof_bdw_ops, SND_SOC_SOF_BROADWELL);
+
 
```

Comment:
Double new lines here? checkpatch should have complained.

Reply:
yes, fixed now, thanks!